docs/features/REST-sphubsite-type.md

```diff
@@ -0,0 +1,24 @@
+---
+title: SPHubSite REST type
```

Comment:
Should be something more like "SP.HubSite Object"--the object type isn't specific to REST.

Reply:
fixed